packages/libs/src/sdk/middleware/addAppInfoMiddleware.ts

```diff
@@ -41,7 +50,7 @@ export const addAppNameMiddleware = ({
 
         appName = (
           await developerApps.getDeveloperApp({
-            address: await services.auth.getAddress()
```

Comment:
this was safe before, because you have to have either an `appName` or an `apiKey`